etl/steps/data/meadow/smoking/2024-05-30/cigarette_sales.py

```diff
@@ -0,0 +1,265 @@
+        "Unnamed: 7",
+        "Unnamed: 8",
+        "Unnamed: 9",
+        "Unnamed: 10",
+        "Unnamed: 11",
+        "Unnamed: 12",
+    ]
+
+    tb_from_excel = Table()
+
+    for cty, cty_sheet in COUNTRY_MAP.items():
+        if cty not in SPECIAL_CASES:
+            # open excel file
+            tb_from_excel = pr.read_excel(
+                zf.open("{}{}".format(folder_name, cty_sheet["file_name"])),
+                metadata=snap_meta,
+                origin=snap.metadata.origin,
+                sheet_name=cty_sheet["sheet_name"],
+                header=9,
+                names=ORIGINAL_COLUMNS,
+            )
+
+        elif cty == "Germany":
+            tb_from_excel = pr.read_excel(
+                zf.open("{}{}".format(folder_name, cty_sheet["file_name"])),
+                sheet_name=cty_sheet["sheet_name"],
+                metadata=snap_meta,
+                origin=snap.metadata.origin,
+                header=2,
+            )
+
+        elif cty == "USSR and fSU":
+            tb_from_excel = pr.read_excel(
+                zf.open("{}{}".format(folder_name, cty_sheet["file_name"])),
+                sheet_name=cty_sheet["sheet_name"],
+                metadata=snap_meta,
+                origin=snap.metadata.origin,
+                header=9,
+            )
+            tb_from_excel["Unnamed: 4"] = "NaN"
+            tb_from_excel["Unnamed: 5"] = "NaN"
+
+        elif cty in FORMAT_SPECIAL_CASES.keys():
+            format = FORMAT_SPECIAL_CASES[cty]
+            tb_from_excel = pr.read_excel(
+                zf.open("{}{}".format(folder_name, cty_sheet["file_name"])),
+                sheet_name=cty_sheet["sheet_name"],
+                metadata=snap_meta,
+                origin=snap.metadata.origin,
+                header=format["header_row"],
+                usecols=format["cols"],
+            )
+            tb_from_excel.columns = [
+                "Year",
+                "Manufactured cigarettes - Millions",
+                "Manufactured cigarettes - Number/adult/day",
+            ]
+            for col in addl_cols:
+                tb_from_excel[col] = "NaN"
+            tb_from_excel = tb_from_excel.head(format["num_rows"])
+
+        # drop empty columns
+        tb_from_excel.drop(labels=["Unnamed: 3", "Unnamed: 6", "Unnamed: 9", "Unnamed: 12"], axis=1, inplace=True)
+
+        # change header and remove sub-header columns
+        tb_from_excel.drop(labels=[0, 1], axis=0, inplace=True)
+        tb_from_excel.columns = NEW_COLUMNS
+
+        tb_from_excel["country"] = cty
+        country_tables.append(tb_from_excel)
+
+    # concatenate tables and delete rows without data
+    smoking_data_tb = pr.concat(country_tables)
+
+    # remove repeating headers
+    smoking_data_tb = smoking_data_tb[smoking_data_tb["year"].notna()]
+    smoking_data_tb = smoking_data_tb[smoking_data_tb["manufactured_cigarettes_millions"].apply(is_not_string)]
+
+    # change data types to string
```

Comment:
It would be good to explain why year needs to be a string (because the most logical choice would be int). Give some examples of the content of "year" in the comment, so that it's clear why you use str.